### ISSUE TITLE
Nvidia Shield Remote keeps taking controller port 1

### ISSUE THREAD

**Issue by nickp85** [libretro/RetroArch#11295]
Device: Nvidia Shield 2017
Version: 9/6 Nightly

The 2019 Nvidia Shield remote keeps getting detected as a controller and assigned to a port. If it does that then my real controller lands in port 2 and I have to force close the app and restart it so that I can make sure my controller is detected first for port 1

**Comment by hizzlekizzle** [comment]
Yes, this is known. Unfortunately, the way the remote functions, we had to hardcode this behavior or it would be entirely impossible to use the remote as an input device at all.

**Comment by nickp85** [comment]
Enhancement then perhaps to make it a configurable option whether or not to ignore the remote?
